app/models/searchgov_domain.rb

```diff
@@ -11,6 +11,20 @@ def delay
     end
   end
 
+  def scheme
+    begin
+      response = open("http://#{domain}/", allow_redirections: :safe, 'User-Agent' => DEFAULT_USER_AGENT)
```

Comment:
@noremmie , I am still half-considering storing this in the `SearchgovDomains` table, but for now I want this checked dynamically. I'll be using the `searchgov_domains.status` column as a kind of kill-switch, meaning that any SearchgovDomain with an error status will be deactivated for things like sitemap indexing, crawling, etc.